config.go

```diff
@@ -121,6 +131,32 @@ type CORSConfig struct {
 	//
 	// Added in v4.2.0.
 	AllowAllOrigins bool
+
+	// AllowOrigins enables CORS and allows the list of origins in the
+	// HTTP request origins when set. Practically speaking, this
+	// results in the HTTP header "AllowOrigins".
+	//
+	// Added in v5.0.0.
+	AllowOrigins string
```

Comment:
Be careful when merging this so it doesn't overwrite the #101 version of the code.